src/component/FetchItems.js

```diff
@@ -9,15 +9,23 @@ import dayjs from 'dayjs';
 const FetchItems = ({ token, setToken, firestore }) => {
   const [empty, setEmpty] = useState(true);
 
+  if (!token) {
+    return <Redirect to="" />;
+  } else {
+    var itemsDocRef = firestore // use var for function scoping
```

Comment:
Non-blocking, but you could (some would say should) use `let` or `const` here instead of `var`

Reply:
Oh wait, I think I see why you're not. You use `itemsDocRef` later. So, using `let` would mean the variable is scoped to the `else` block. The correct thing to do here is to define the variable outside (before) the `if`/`else` block and assign it a value within `else`. The reason is that it's technically not correct to declare a variable with `var` within a block like this. Arcane JavaScript weirdness!

Reply:
You'll find that in order to declare the variable in one place and assign a value in another place, you'll need to use `let` ← hopefully that saves you some time!

Reply:
Ya know, I thought about that but went for `var` to do it all on one line 🙃 
Heh - I'll change it over to `let`!

Reply:
Thanks @segdeha! 😁